dataservice/api/sample/models.py

```diff
@@ -18,6 +18,7 @@ class Sample(db.Model, Base):
             acquired, expressed in number of days since birth
     :param tumor_descriptor: The kind of disease present in the tumor
            specimen as related to a specific timepoint
+    :param uberon: Uber-anatomy ontology for anatomical_site
```

Comment:
Same comment as before. If this is supposed to be the id of the term in Uberon, then it should probably be named `uberon_id`.

Reply:
👍